examples/example-revenue-report/src/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": "1",
+  "information": {
+    "name": "example-revenue-report",
+    "displayName": "[Example] Revenue report",
+    "description": "This is a sample UCRM plugin showcasing a custom link in menu and security checks.",
+    "url": "https://github.com/Ubiquiti-App/UCRM-plugins",
```

Comment:
URL should point to plugin's root, so that the README is visible when you open the link.

Reply:
```suggestion
    "url": "https://github.com/Ubiquiti-App/UCRM-plugins/examples/example-revenue-report",
```